openslides_backend/action/actions/motion/create.py

```diff
@@ -178,3 +179,42 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         )
 
         return instance
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        # Check can create amendment if needed.
+        if instance.get("lead_motion_id"):
+            perm = Permissions.Motion.CAN_CREATE_AMENDMENTS
+            if not has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
+                msg = f"You are not allowed to perform action {self.name}."
+                msg += f" Missing permission: {perm}"
+                raise PermissionDenied(msg)
+
+        # general check can create
+        perm = Permissions.Motion.CAN_CREATE
+        if not has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
+            msg = f"You are not allowed to perform action {self.name}."
+            msg += f" Missing permission: {perm}"
+            raise PermissionDenied(msg)
+
+        # if not can manage whitelist the fields.
+        perm = Permissions.Motion.CAN_MANAGE
+        if not has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
+            whitelist = [
+                "title",
+                "text",
+                "reason",
+                "lead_motion_id",
+                "amendment_paragraph_$",
+                "category_id",
+                "statute_paragraph_id",
+                "workflow_id",
+                "id",
+                "meeting_id",
+            ]
+            if instance.get("lead_motion_id"):
+                whitelist.append("motion_block_id")
+            for field in instance:
+                if field not in whitelist:
+                    msg = f"You are not allowed to perform action {self.name}."
+                    msg += f" Missing permission: {perm}"
+                    raise PermissionDenied(msg)
```

Comment:
Right, but we explicitly want to throw errors for not-accepted fields. We do not want to silently drop things, so the user thinks it was accepted, but just ignored.

Reply:
So raising an exception is the right thing.

Reply:
Ok, so we change our exception-paradigm compared to OS3. Noted.

Reply:
Regarding the message, I would then prefer to list the forbidden fields instead of the missing permission since that was the deciding factor. What do you think?

Reply:
Mmh, not sure, I have changed the code in create and update. Perhaps this is better.